install_config/persistent_storage/topics/glusterfs_example_full.adoc

```diff
@@ -3,25 +3,36 @@
 ----
 [OSEv3:vars]
 ...
-openshift_registry_selector="role=infra"                  <1>
-openshift_hosted_registry_storage_kind=glusterfs
+openshift_hosted_registry_storage_kind=glusterfs <1>
+openshift_hosted_registry_storage_volume_size=5Gi
+openshift_hosted_registry_selector='node-role.kubernetes.io/infra=true'
 
-openshift_metrics_hawkular_nodeselector={"role":"infra"}  <1>
-openshift_metrics_cassandra_nodeselector={"role":"infra"} <1>
-openshift_metrics_heapster_nodeselector={"role":"infra"}  <1>
+openshift_metrics_install_metrics=true
+openshift_metrics_hawkular_nodeselector={"node-role.kubernetes.io/infra": "true"} <1>
+openshift_metrics_cassandra_nodeselector={"node-role.kubernetes.io/infra": "true"} <1>
+openshift_metrics_heapster_nodeselector={"node-role.kubernetes.io/infra": "true"} <1>
 openshift_metrics_storage_kind=dynamic
```

Comment:
Add this option 
openshift_metrics_storage_volume_size=10Gi

Reply:
Oh, yes! Thanks for catching that.